New version: Boscia v0.1.23

JuliaRegistrator opened the pull request:
- Registering package: Boscia
- Repository: https://github.com/ZIB-IOL/Boscia.jl
- Created by: @dhendryc
- Version: v0.1.23
- Commit: 5edfb0d4db436b4e1dfe15887a3e1036fa750adb
- Reviewed by: @dhendryc
- Reference: https://github.com/ZIB-IOL/Boscia.jl/commit/5edfb0d4db436b4e1dfe15887a3e1036fa750adb#commitcomment-140011970
- Description: Mixed-Integer Convex Programming: Branch-and-bound with Frank-Wolfe-based convex relaxations
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6a41893bb55f0a6d93928f2bbc0db748305c09f9b93d674172f81cccd1de9527133d2cda8e502d8a9479e6822f2b0546a08d50adc6b5c24573da3160e83850d82df364c2f01bdac8011585a6b0f342f7b3d43722d63668c13b08ea952f91dcc61bc56ad609e83ef5dd76792c6b1457687344fcd53bef7e12dfc0fbb2205fa1f0c602dd8c697ba47843c3c1e6cd56ac94f84acabe8f4060feefbd22d460e02e31009fb457ecc8a62abdc2110ffb48e3f1a -->

matbesancon commented:
[noblock]
Does someone know what the issue can be? The error in CI is a bit cryptic:
```
┌ Info: Version can be `import`ed
│   meets_this_guideline = true
└   message = ""
ERROR: LoadError: RegistryCI.AutoMerge.AutoMergeGuidelinesNotMet("The automerge guidelines were not met.")
Stacktrace:
 [1] pull_request_build(data::RegistryCI.AutoMerge.GitHubAutoMergeData; check_license::Bool)
   @ RegistryCI.AutoMerge ~/.julia/packages/RegistryCI/4CUkB/src/AutoMerge/pull_requests.jl:253
 [2] pull_request_build
   @ ~/.julia/packages/RegistryCI/4CUkB/src/AutoMerge/pull_requests.jl:173 [inlined]
 [3] pull_request_build(api::GitHub.GitHubWebAPI, pr_number::Int64, current_pr_head_commit_sha::String, registry::GitHub.Repo, registry_head::String; whoami::String, auth::GitHub.OAuth2, authorized_authors::Vector{String}, authorized_authors_special_jll_exceptions::Vector{String}, error_exit_if_automerge_not_applicable::Bool, master_branch::String, master_branch_is_default_branch::Bool, suggest_onepointzero::Bool, point_to_slack::Bool, registry_deps::Vector{String}, check_license::Bool, public_registries::Vector{String}, read_only::Bool, environment_variables_to_pass::Vector{String})
   @ RegistryCI.AutoMerge ~/.julia/packages/RegistryCI/4CUkB/src/AutoMerge/pull_requests.jl:168
 [4] pull_request_build
   @ ~/.julia/packages/RegistryCI/4CUkB/src/AutoMerge/pull_requests.jl:72 [inlined]
 [5] run(; env::Base.EnvDict, cicfg::RegistryCI.AutoMerge.GitHubActions, merge_new_packages::Bool, merge_new_versions::Bool, new_package_waiting_period::Day, new_jll_package_waiting_period::Minute, new_version_waiting_period::Minute, new_jll_version_waiting_period::Minute, registry::String, tagbot_enabled::Bool, authorized_authors::Vector{String}, authorized_authors_special_jll_exceptions::Vector{String}, additional_statuses::Vector{String}, additional_check_runs::Vector{String}, error_exit_if_automerge_not_applicable::Bool, master_branch::String, master_branch_is_default_branch::Bool, suggest_onepointzero::Bool, point_to_slack::Bool, registry_deps::Vector{String}, api_url::String, check_license::Bool, public_registries::Vector{String}, read_only::Bool, environment_variables_to_pass::Vector{String})
   @ RegistryCI.AutoMerge ~/.julia/packages/RegistryCI/4CUkB/src/AutoMerge/public.jl:144
 [6] top-level scope
   @ ~/work/_temp/2c4afb7e-c469-4b56-ae2a-e681c84159fb:4
in expression starting at /home/runner/work/_temp/2c4afb7e-c469-4b56-ae2a-e681c84159fb:4
```

JuliaRegistrator committed:
New version: Boscia v0.1.23

UUID: 36b166db-dac5-4d05-b36a-e6c4cef071c9
Repo: https://github.com/ZIB-IOL/Boscia.jl.git
Tree: 7796a38609293ae95077d54d0835707c02b70930

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
